deployment/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM java:8-jre-alpine
```

Comment:
This is generated from lein new onyx-app template. Not sure what the knock on would be if we changed it.